src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const message = (className, text) => {
```

Comment:
```suggestion
const showMessage = (className, text) => {
```
![image](https://user-images.githubusercontent.com/96409563/212742270-7aae8940-e8f5-4300-949e-7f8ee8bd3dfb.png)